README.md

```diff
@@ -255,36 +255,37 @@ In diesen Ordner wird die __Datenbank__ des Containers _persistiert_, also dauer
+The Dockerfiles for Mailhog and MySQL are quite simple, because they contain only the indication, which image is used, without this then further adapted. The PHP Apache Dockerfile is more elaborate: Here we first determine the image, but send some adjustments afterwards. For example, we enable Apache modules and install PHP extensions that REDAXO needs. Afterwards we check, if our Webroot - more about that! - is still empty, and if it is, we get a fresh REDAXO from GitHub and unpack it into the webroot.
 
-Die anderen Dateien enthalten Setup-Skripte, Konfigurationen für PHP, Apache und die Datenbank.
+The other files contain setup scripts, configurations for PHP, Apache, and the database.
 
 #### Webroot
 
     html/
 
-Dieses Verzeichnis bildet den __Webroot__, der oben bereits genannt wurde. Es ist verknüpft mit dem Verzeichnis des Containers (ein Debian GNU/Linux übrigens), in dem der Apache-Webserver die Website hinterlegt. Wenn du also Anpassungen am REDAXO vornimmst, stehen diese unmittelbar dem Server zur Verfügung, und ebenso andersrum.  
-Das bedeutet: Ebenso wie die Datenbank liegt dein REDAXO dauerhaft auf deinem System und kann von dir bearbeitet werden, während Docker dir nur die notwendige Serverumgebung bereitstellt.
+This directory forms the __Webroot__, which was already mentioned above. It is linked to the directory of the container (a Debian GNU / Linux by the way) in which the Apache web server deposits the website. So if you make adjustments to the REDAXO, they are immediately available to the server, and vice versa.
+This means: Like the database, your REDAXO is permanently on your system and can be edited by you while Docker only provides you with the necessary server environment.
 
-:point_right: _Beachte: Wenn der Ordner beim Start des Containers leer ist, installiert Docker ein frisches REDAXO für dich, und je nach Konfiguration (in `docker-compose.yml`) sogar noch eine Website-Demo dazu. Enthält der Ordner aber bereits Inhalte, ändert Docker nichts daran und startet lediglich den Container._
+
+:point_right: _Keep in mind: If the folder is empty at the start of the container, Docker will install a fresh REDAXO for you, and depending on the configuration (in `docker-compose.yml`) even a website demo. But if the folder already contains content, Docker does not change it and just starts the container._
 
 #### Ignore
 
     .dockerignore
 
-In [dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file) wird definiert, welche Dateien und Ordner _nicht_ an den Docker-Daemon überreicht werden. Wenn dein Projektordner sehr voll ist, kannst du die für Docker unwichtigen Daten übergehen und sparst damit Ressourcen.
+In [dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file) defines which files and folders _not_ will be given to the Docker daemon. If your project folder is very busy, you can bypass Docker's unimportant data and save resources.
```

Comment:
The File [dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file) defines which files and folders will _not_ be given to the Docker daemon. If your project folder is very busy, you can bypass Docker's unimportant data and save resources.